### ISSUE TITLE
DOTA 2 general protection fault - every time

### ISSUE THREAD

**Issue by dgiorgio** [ValveSoftware/Dota-2#2095]
#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/): 
https://gist.github.com/dgiorgio/26ca65451881fe55cc88492f55ae1eba

#### Please describe your issue in as much detail as possible:
the game closes unexpectedly.
dmesg
```
[sáb jun 18 12:58:19 2022] show_signal: 55 callbacks suppressed
[sáb jun 18 12:58:19 2022] traps: GlobPool/4[5927] general protection fault ip:7f637e534b34 sp:7f6398b793e8 error:0 in libanimationsystem.so[7f637e000000+65c000]
[sáb jun 18 13:03:07 2022] dota2[11734]: segfault at 7ffddd9945b0 ip 00007ffddd9945b0 sp 00007ffddd9945a0 error 15
[sáb jun 18 13:03:07 2022] Code: 00 00 cc a7 72 62 93 7f 00 00 e7 09 35 b8 f8 ad e1 bd b0 45 99 dd fd 7f 00 00 0a 00 00 00 00 00 00 00 3f 00 00 e0 00 00 00 00 <c0> 45 99 dd fd 7f 00 00 37 4d 73 62 93 7f 00 00 70 66 99 dd fd 7f
[sáb jun 18 16:07:21 2022] traps: GlobPool/1[29710] general protection fault ip:7fe9e7534b34 sp:7fea01dfa1a8 error:0 in libanimationsystem.so[7fe9e7000000+65c000]
```
I have the dmp files, where do I send?

#### Steps for reproducing this issue:

1. just play
2. 
3.

**Comment by kisak-valve** [comment]
Hello @dgiorgio, are you still experiencing this issue on an up to date system?

**Comment by kisak-valve** [comment]
Most likely fixed along with #2065. Closing as a duplicate.
